README.md

```diff
@@ -249,13 +218,119 @@ Invoking the `eval` method of each dataset yields all metrics reported in the pa
+| globalwheat     | Image    | 10                 | 10                | 2                       |
+| civilcomments   | Text     | 0.1                | 0.3               | 4.5                     |
+| fmow            | Image    | 50                 | 55                | 6                       |
+| poverty         | Image    | 12                 | 14                | 5                       |
+| amazon          | Text     | 7                  | 7                 | 5                       |
+| py150           | Text     | 0.1                | 0.8               | 9.5                     |
+
+The following are the sizes of the unlabeled data bundles:
+
+| Dataset command | Modality | Download size (GB) | Size on disk (GB) |
+| --------------- | -------- | ------------------ | ----------------- |
+| iwildcam        | Image    | 41                 | 41                |
+| camelyon17      | Image    | 69.4               | 96                |
+| ogb-molpcba     | Graph    | 1.2                | 21                |
+| globalwheat     | Image    | 103                | 108               |
+| civilcomments   | Text     | 0.3                | 0.6               |
+| fmow\*          | Image    | 50                 | 55                |
+| poverty         | Image    | 172                | 184               |
+| amazon\*        | Text     | 7                  | 7                 |
+
+\* The unlabeled dataset is the same as the labeled dataset.
+
+While the `camelyon17` dataset is small and fast to train on, we advise against using it as the only dataset to prototype methods on, as the test performance of models trained on this dataset tend to exhibit a large degree of variability over random seeds.
+
+The image datasets (`iwildcam`, `camelyon17`, `rxrx1`, `globalwheat`, `fmow`, and `poverty`) tend to have high disk I/O usage. If training time is much slower for you than the approximate times listed above, consider checking if I/O is a bottleneck (e.g., by moving to a local disk if you are using a network drive, or by increasing the number of data loader workers). To speed up training, you could also disable evaluation at each epoch or for all splits by toggling `--evaluate_all_splits` and related arguments.
+
+### Algorithms
+In the `examples/algorithms` folder, we provide implementations of the adaptation algorithms benchmarked in our papers ([1](https://arxiv.org/abs/2012.07421), [2](https://arxiv.org/abs/2112.05090)).
+All algorithms train on labeled data from a WILDS dataset's _train_ split.
+Some algorithms are designed to also leverage unlabeled data. To load unlabeled data, specify an `--unlabeled_split` when running.
+
+| Algorithm command                                | Hyperparameters                                                                            | Notes                             | See WILDS paper                                                                |
```

Comment:
Maybe say these are algorithm-specific hyperparameters, and that one can also tune things like LR, weight decay, and labeled/unlabeled batch size?